docker-compose.yml

```diff
@@ -36,12 +36,11 @@ services:
   web3:
     extends:
       service: web
+    image: addons/addons-server-py3
```

Comment:
We need to create this on docker hub once this PR is merged, have it point to this repo using `Dockerfile.python3`

Reply:
Alternatively, to test this PR locally, fetch the code and:
`docker-compose stop web3`
`docker build -f Dockerfile.python3 -t addons/addons-server-py3 .`
`docker-compose up -d web3` 

The `web3` container should now be the new container with Python 3.6.